### PR TITLE
PMM-9590 credentials source flag

### PR DESCRIPTION
**Pull request by rasika-chivate**
Added --credentials-source option to pmm-admin.

### CODE REVIEW

File: docs/details/commands/pmm-admin.md [hunk 1]
```diff
@@ -271,6 +273,22 @@ When you remove a service, collected data remains on PMM Server for the specifie
...
+
+#### `pmm-admin add --credentials-source=CREDENTIALS-SOURCE [FLAGS] [NAME] [ADDRESS]`
+:     Pass senstive information such as username and password via a JSON file instead of exposing this information in the existing flags `--username`, `--password` and `--agent-password`.
+
+    ` ----credentials-source=CREDENTIALS-SOURCE`
+    : Path to the JSON file where the credentials are stored.
+    
+    Example
+
+    `pmm-admin add mysql --credentials-source=/myfolder/pmm-credentials.json`
```
Comment (denisok) [inline docs/details/commands/pmm-admin.md:286]: please add json file example

**Review by denisok — CHANGES_REQUESTED** [review]
this are new changes - they suppose to go to 2.29.0 not in `main`

File: docs/details/commands/pmm-admin.md [hunk 2]
```diff
@@ -14,6 +14,8 @@
 
 DATABASE:= [[MongoDB](#mongodb) | [MySQL](#mysql) | [PostgreSQL](#postgresql) | [ProxySQL](#proxysql)]
 
+`pmm-admin add --credentials-source=CREDENTIALS-SOURCE [FLAGS] [NAME] [ADDRESS]`
```
Comment (beatahandzelova) [inline docs/details/commands/pmm-admin.md:17]: The flag --credentials-source is available also with pmm-admin add external/external-serverless and haproxy, maybe it's better to move it to flags under every db type section?

**Comment by rnovikovP** [comment]
⚠️  the PMM-9590  is for  2.29 so the  Doc needs to be 2.29 as if we publish it's earlier - this will not  work until 2.29 released 
(or we already changed the Doc schema ...)

**Comment by rasika-chivate** [comment]
> ⚠️ the [PMM-9590](https://jira.percona.com/browse/PMM-9590) is for 2.29 so the Doc needs to be 2.29 as if we publish it's earlier - this will not work until 2.29 released (or we already changed the Doc schema ...)

@rnovikovP I will move it to 2.29.0

**Pull request closed without merging**